gf127/gf127.go

```diff
@@ -12,7 +12,8 @@ import (
 type GF127 [2]uint64
 
 const (
-	byteSize  = 16
+	// ByteSize shows GF127 state size in bytes.
+	ByteSize  = 16
```

Comment:
Can it be just `Size`? We have `sha256.Size`, `md5.Size`, likely we can have `gf127.Size` as well (even though it's not a complete hash).

Reply:
Sure, updated